Assets/Scripts/SS3D/Systems/Furniture/UnlockLockerInteraction.cs

```diff
@@ -0,0 +1,70 @@
+﻿using SS3D.Data;
+using SS3D.Data.Enums;
+using SS3D.Interactions;
+using SS3D.Interactions.Extensions;
+using SS3D.Interactions.Interfaces;
+using SS3D.Systems.Furniture;
+using SS3D.Systems.Inventory.Containers;
+using SS3D.Systems.Inventory.Items;
+using System;
+using UnityEngine;
+
+namespace SS3D.Systems.Inventory.Interactions
+{
+    public sealed class UnlockLockerInteraction : Interaction
```

Comment:
Can't this be made more generic? UnlockInteraction?

Reply:
This is only for testing Permission Checking, it should be removed once we add actual lockers and airlocks

Reply:
Add a comment indicating that so others know.